packages/jellyfish-transaction/package.json

```diff
@@ -38,7 +38,6 @@
     "bignumber.js": "^9.0.1"
   },
   "dependencies": {
-    "@defichain/jellyfish-crypto": "0.0.0",
     "smart-buffer": "^4.1.0"
   },
   "devDependencies": {
```

Comment:
I think we can remove all `devDependencies`, since we end up not using it anyway.